shell/package.json

```diff
@@ -16,15 +16,11 @@
     "clean": "./scripts/clean",
     "lint": "./node_modules/.bin/eslint --max-warnings 0 --ext .ts,.js,.vue .",
     "test": "./node_modules/.bin/nyc ava --serial --verbose",
-    "nuxt": "./node_modules/.bin/nuxt",
-    "dev": "./node_modules/.bin/nuxt dev",
-    "mem-dev": "node --max-old-space-size=8192 ./node_modules/.bin/nuxt dev",
+    "dev": "./node_modules/.bin/vue-cli-service dev",
```

Comment:
In nuxt I had to use mem-dev otherwise I couldn't build our dashboard. I no longer needed mem-dev to so I removed this. If we want to add it back we can but it seems like we should get rid of things that are no longer necessary.